src/common/Math.h

```diff
@@ -27,6 +27,11 @@ bool IsAligned(const void* ptr, size_t alignment);
 void* AlignVoidPtr(void* ptr, size_t alignment);
 
 template<typename T>
+T Align(T value, size_t alignment) {
+    return static_cast<T>(reinterpret_cast<uintptr_t>(AlignVoidPtr(reinterpret_cast<void*>(static_cast<uintptr_t>(value)), alignment)));
```

Comment:
I don't think it's valid to have a templated and non-templated function both called the same thing. We could name one AlignNumber or AlignPtr?